src/Tree/Tree.js

```diff
@@ -1,238 +1,330 @@
-import { Dropdown } from '../Dropdown/Dropdown';
+import classnames from 'classnames';
 import PropTypes from 'prop-types';
+import shortid from 'shortid';
 import React, { Component } from 'react';
 
-export class Tree extends Component {
+export class TreeCell extends Component {
```

Comment:
I believe that Treeview and SideNav have a lot in common.

Let's try and standardize on naming conventions for subcomponents, where it makes sense.

Reply:
I agree. I think the `Tree` component has one extra layer though for the nested elements, as well as the header elements. What do you think of:
```
<Tree>
    <TreeHeader>
        <TreeListItem>
     <TreeList>
         <TreeRow>
            <TreeListItem>
```
Where `SideNav` is:
```
<SideNav>
    <SideNavList>
        <SideNavListItem>
```

Reply:
Correct, it doesn't look like you should ever have more than one `TreeHeader`